### PR TITLE
Arcvancouver vancouverbc

### PR DESCRIPTION
**Pull request by arcvancouver**
Leave a comment

### CODE REVIEW

File: arcvancouver-vancouverbc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+### Vancouver BC Canada
+
+### Food  :burgers: 
+ - McD's
+ - sushi
+ - 
+### Things to Do :eyeroll:
+ - Fireworks 
+ - Grouse Grind
```
Comment (realslimshanky) [inline arcvancouver-vancouverbc.md:10]: @arcvancouver What's that ?

File: arcvancouver-vancouverbc.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+### Vancouver BC Canada
+
+### Food  :burgers: 
+ - McD's
```
Comment (anwoyy) [inline arcvancouver-vancouverbc.md:5]: I love Sushi as well :)

File: arcvancouver-vancouverbc.md (hunk 1 above)
Comment (arcvancouver) [inline arcvancouver-vancouverbc.md:10]: trail around the largest park downtown

File: arcvancouver-vancouverbc.md (hunk 2 above)
Comment (arcvancouver) [inline arcvancouver-vancouverbc.md:5]: So cheap! :+1:

**Pull request closed without merging**